test_deployment/openshift.deploy.yml

```diff
@@ -0,0 +1,175 @@
+apiVersion: template.openshift.io/v1
```

Comment:
Please use the same templates across DEV, TEST and PROD.  Use parameters to handle different variables.